mica/stats/update_acq_stats.py

```diff
@@ -607,7 +609,7 @@ def calc_stats(obsid):
         elif obsid in [60983]:
             starcheck = get_starcheck_catalog(obsid, mp_dir="/2002/OCT2102/oflsc/")
         elif obsid in [60640, 60639, 60638, 60637, 60636, 60635, 60634, 60633]:
-            raise ValueError("Starcheck not available for PCHECK_JUL2003")
+            raise ValueError("Starcheck not available for PCHECK_JUL2003") from err
```

Comment:
Did you consider raise from None?

Reply:
Just checked the code, and this one should definitely be `from None`

Reply:
What makes this a "definitely" for you?  I see it as being more ambiguous and pretty benign either way.  One issue for this code area is that the code in the "try/except" doesn't fail on these obsids since the change to kadi commands for the lookups.  So the ValueError isn't hit in the first place.  If I raise an exception in the try block:
```
try:
        starcheck = get_starcheck_catalog_at_date(manvr.acq_start)
        raise ValueError("Fail on purpose")
``` 
I can of course see that if I raise from _err_ and don't end up handling the exception in calling code I get the double stacktrace and if I raise from _None_ I don't.

Double stacktrace:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
File /proj/sot/ska/jeanproj/git/mica/mica/stats/update_acq_stats.py:591, in calc_stats(obsid)
    590     starcheck = get_starcheck_catalog_at_date(manvr.acq_start)
--> 591     raise ValueError("Fail on purpose")
    592 except Exception as err:
    593     # No matching observations for some known observations with problems. Use
    594     # hard-coded input for get_starcheck_catalog.

ValueError: Fail on purpose

The above exception was the direct cause of the following exception:

ValueError                                Traceback (most recent call last)
Cell In[6], line 1
----> 1 mica.stats.update_acq_stats.calc_stats(60640)

File /proj/sot/ska/jeanproj/git/mica/mica/stats/update_acq_stats.py:608, in calc_stats(obsid)
    606     starcheck = get_starcheck_catalog(obsid, mp_dir="/2002/OCT2102/oflsc/")
    607 elif obsid in [60640, 60639, 60638, 60637, 60636, 60635, 60634, 60633]:
--> 608     raise ValueError("Starcheck not available for PCHECK_JUL2003") from err
    609 elif obsid in [60616, 60615]:
    610     starcheck = get_starcheck_catalog(obsid, mp_dir="/2003/JUL2103/oflsc/")

ValueError: Starcheck not available for PCHECK_JUL2003
```

Single stacktrace:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[7], line 1
----> 1 mica.stats.update_acq_stats.calc_stats(60640)

File /proj/sot/ska/jeanproj/git/mica/mica/stats/update_acq_stats.py:608, in calc_stats(obsid)
    606     starcheck = get_starcheck_catalog(obsid, mp_dir="/2002/OCT2102/oflsc/")
    607 elif obsid in [60640, 60639, 60638, 60637, 60636, 60635, 60634, 60633]:
--> 608     raise ValueError("Starcheck not available for PCHECK_JUL2003") from None
    609 elif obsid in [60616, 60615]:
    610     starcheck = get_starcheck_catalog(obsid, mp_dir="/2003/JUL2103/oflsc/")

ValueError: Starcheck not available for PCHECK_JUL2003
```

I think if the exception ends up unhandled in calling code that the double stack trace is more useful.  But I'm not passionate about this.